upgrade.md

```diff
@@ -67,18 +67,27 @@ You should update the following dependencies in your application's `composer.jso
 <div class="content-list" markdown="1">
 
 - `laravel/framework` to `^11.0`
-- `laravel/sanctum` to `^4.0`
+- `laravel/sanctum` to `^4.0` (If installed)
 - `laravel/telescope` to `^5.0` (If installed)
+- `laravel/passport` to `^12.0` (If installed)
```

Comment:
I don't think we need `(If installed)` here next to these packages since it's a bit of a given to only do this if you have these packages installed.